enterprise/utils.py

```diff
@@ -1564,7 +1628,7 @@ def enroll_licensed_users_in_courses(enterprise_customer, licensed_users_info, d
         user = User.objects.filter(email=licensed_user_info['email']).first()
         try:
             if user:
-                succeeded = enroll_user(enterprise_customer, user, course_mode, course_run_key)
```

Comment:
not refactoring the `enroll_user` function in this PR since I find that Manage learners also uses that function and it is going to risk adding regressions there. I want to remove the use of enroll_user in favor of this, but I also don't want to delay bulk enrollment. Also a lot of tests are written for manage learners that need updates. I would rather do that in a new PR. And we have many places to replace the enrollmentApiClient anyway

Reply:
@binodpant I think this is fine, when you make the new ticket can you link it here?